_layouts/wiki.html

```diff
@@ -15,14 +15,6 @@
     <div id="maincontainer" class="fx-col-100-xs fx-col-50-m">
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="ftext" class="fx-txt-center">
-        {% capture footertext %}
-        {% if site.active_lang %}
-        {% include wiki/{{ site.active_lang }}/footertext.md %}
-        {% endif %}
-        {% endcapture %}
-        {{ footertext | markdownify }}
-      </div>
```

Comment:
If this is removed, the respective CSS should also be removed. But I don't quite understand why we need to remove it.

Reply:
I don't know enough about the CSS to remove that CSS. The reason I removed it was so that we just have one footer on the website, don't confuse CC with ASIO and Jamulus rights, and to close the survey which needs removing now until I get round to doing a better one.